packages/components/src/code-block/view/components/copy-button.tsx

```diff
@@ -0,0 +1,106 @@
+import { defineComponent, h, Fragment, ref } from 'vue'
+
+import { Icon } from '../../../__internal__/components/icon'
+
+h
+Fragment
+
+export type CopyButtonProps = {
+  copyText: string
+  copiedText: string
```

Comment:
I think we don't need this `copiedText` in UI. This will make the button shifting.
We should provide a `onCopy` callback instead.

https://github.com/Milkdown/milkdown/blob/4e27fdadd6bc21eab3292cd09074bb351d5959f5/packages/components/src/link-tooltip/slices.ts#L55

This will allow the developers to call a toast to notify the users.